Microsoft.Alm.Authentication/Microsoft.Alm.Authentication.csproj

```diff
@@ -59,13 +59,15 @@
       <HintPath>..\packages\Microsoft.IdentityModel.Clients.ActiveDirectory.2.18.206251556\lib\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.WindowsForms.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="PresentationFramework" />
     <Reference Include="System" />
     <Reference Include="System.Core" />
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="System.Data" />
     <Reference Include="System.Drawing" />
     <Reference Include="System.Net.Http" />
     <Reference Include="System.Security" />
+    <Reference Include="WindowsBase" />
```

Comment:
The `Microsoft.Alm.Authentication` library cannot have a dependency on a `WindowsBase`. I actually forget the reason, but it's on my short-list of things not to depend on. :sheep: